syft/federated/train_config.py

```diff
@@ -23,14 +23,13 @@ def __init__(
         owner: workers.AbstractWorker = None,
         batch_size: int = 32,
         epochs: int = 1,
-        optimizer: str = "sgd",
-        lr: float = 0.1,
+        optimizer: str = "SGD",
+        optimizer_args: dict = {"lr": 0.1, "weight_decay": 0.01},
```

Comment:
I would tend to setting the weight_decay by default to None.

Reply:
The weight decay was originally defaulted to 0.01 (before this pr). I am not opposed to setting it to None (in this case just not included) but I wanted to maintain the original behavior (with that in mind, what is your preference)

Reply:
I vote for None.

Reply:
I added the weight decay and I also vote for None (default value of the sgd optimizer). That's how I thought I left it.